src/tracing/errors.test.ts

```diff
@@ -19,4 +19,11 @@ describe('filterKnownErrors', () => {
     const originalException = new Error('underlying network changed')
     expect(filterKnownErrors(ERROR, { originalException })).toBe(null)
   })
+
+  it('filters call revert errors', () => {
+    const originalException = new Error(
```

Comment:
need to look into how to make this match as an ethers exception

Reply:
I believe you can reuse the logic from the prior test, with `method: 'collect(...)'`.